app/code/core/Mage/Adminhtml/Block/System/Config/Form/Fieldset/Modules/DisableOutput.php

```diff
@@ -49,6 +39,22 @@ public function render(Varien_Data_Form_Element_Abstract $element)
         return $html;
     }
 
+    public function getModules(): array
+    {
+        $modules = (array)Mage::getConfig()->getNode('modules')->children();
```

Comment:
Won't this break BC? Before this PR:
```php
$modules = array_keys((array)Mage::getConfig()->getNode('modules')->children());
```
It is a simple array with int keys. 

After this PR, the keys are string, 'Mage_Core' etc. It may break 3rd-party observer.

Reply:
Yes, it would be. [But] I think the observer was useles before. Will change it.